docs/node_scenarios.md

```diff
@@ -32,6 +32,10 @@ After creating the service account you'll need to enable the account using the f
 
 Use 'generic' or do not add the 'cloud_type' key to your scenario if your cluster is not set up using one of the current supported cloud types
 
+#### OPENSTACK
+
+**NOTE**: For clusters with OPENSTACK Cloud, make sure [OPENSTACK CLI](pip3 install python-openstackclient) is installed and then create and source the [OPENSTACK RC file](https://docs.openstack.org/newton/user-guide/common/cli-set-environment-variables-using-openstack-rc.html) from the server where Kraken runs.
```

Comment:
I think we can add the openstack python client to the requirements

Reply:
Also, can we move this section up above line 31 so all the clouds are together and maybe add another line in-between the end of openstack and the current line 31.